.github/workflows/upload.yml

```diff
@@ -0,0 +1,163 @@
+name: Upload
+on:
+  release:
+    types: [published]
+
+env:
+  TF_VERSION: 2.6
+  TORCH_VERSION: 1.10.0+cpu
+  COVERAGE_FLAGS: "--cov=pennylane --cov-report=term-missing --cov-report=xml --no-flaky-report -p no:warnings --tb=native"
+
+
+jobs:
+
+  core-tests:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        config:
+          - { python-version: 3.7, interfaces: [ 'torch' ] }
+          - { python-version: 3.7, interfaces: [ 'tf' ] }
+          - { python-version: 3.8, interfaces: [ 'tf', 'torch', 'jax' ] }
```

Comment:
Should these be split up as well, or the 3.7 be combined into one?

Reply:
It seems that we had some issues that were only unearthed when multiple interfaces were installed. This one is copied from the original test suite yaml file.